app/controllers/lookups_controller.rb

```diff
@@ -1,7 +1,12 @@
 class LookupsController < ApplicationController
   def by_base_path
+    # return content_ids for content that is visible on the live site
+    # withdrawn items are still visible
+    states = %w(published unpublished)
```

Comment:
Not sure if I'm following everything here, but I'd like the behaviour of this endpoint to be the same as if you're looking up a content_id from the site by checking the _live_ content-store (https://www.gov.uk/passport-fees -> https://www.gov.uk/api/content/passport-fees -> `8b59b474-9775-4366-97ee-97a66740411c`). 

Is the `%w[published withdrawn]` filter enough for that?